src/main/java/MinesweeperController.java

```diff
@@ -255,8 +257,48 @@ public void handle(MouseEvent mouseEvent) {
         }
     }
 
+    public void playBestStrategy(){
```

Comment:
A javadoc comment before this method explaining what it is is a good idea since it's not clear by the name what this method does.